leaderboardsgg/leaderboard-site
# Package bumps & usage of ts 4.9 Satisfies operator

## What

- Bumped all packages to latest and tested to make sure nothing weird happened from that
- Added usage of TS 4.9+ `satisfies` operator to add object shape checking without changing the type when it's being used

### Steps for testing

Standard testing check to see the dialogs and everything renders and the Vitest tests run
